lib/pbench/cli/agent/core.py

```diff
@@ -0,0 +1,50 @@
+"""
+Base classes for various agent cli commands
+
+Base classes are used so that code can be shared between the same type
+of commands without duplicating the same code everywhere.
+"""
+
+from pbench.agent.base import BaseCommand
+
+
+class ToolCommand(BaseCommand):
+    """Common tool command Class"""
+
+    def __init__(self, context):
+        super().__init__(context)
+
+
+class LogCommand(BaseCommand):
```

Comment:
Is there a reason that we need all the command classes in the same file?  Why not have a `base.py` and a `tool.py`, `log.py`, etc?  If there is a good reason to have them all in the same file, great; however, it is not obvious why they need to be.

Having them in separate files means that we can add and convert commands as necessary without merge conflicts as we work.